tests/test_fastapi/test_binary_fields.py

```diff
@@ -37,7 +38,7 @@ async def shutdown() -> None:
         await database_.disconnect()
 
 
-blob3 = b"\xc3\x28"
+blob3 = b"\xc3\x83\x28"
```

Comment:
why this change?

Reply:
The utf was invalid, there is no c3 or c3 28.
It was failing on trying to decode it because of that.
C3 83 is greek capital A 
https://www.cogsci.ed.ac.uk/~richard/utf-8.cgi?input=00C2&mode=hex